web-console/4.0-designs/storage/Expand Storage Cluster/Expand Storage Cluster.md

```diff
@@ -0,0 +1,74 @@
+# Expand OCS Cluster
+
+## Supported Deployments for OCS 4.2
+
+- AWS (IPI & UPI)
+- VMware (UPI only)
+- RHHI.Next (Bare metal)
```

Comment:
Should this be "Bare Metal (UPI only)"?

Reply:
All baremetal is UPI, but the design for OCS 4.2 is not pertaining to RHHI.Next nor Bare Metal.